test/components/link.spec.js

```diff
@@ -115,17 +116,17 @@ const immutableLinkTest = {
             if (action.type === REPLACE) {
               const { payload } = action;
               captureErrors(done, () => {
-                expect(payload).to.have
-                  .property('pathname')
+                expect(payload)
+                  .to.have.property('pathname')
                   .that.contains('/home/messages/a-team');
 
                 if (typeof href === 'string') {
-                  expect(payload).to.have
-                    .property('search')
+                  expect(payload)
+                    .to.have.property('search')
```

Comment:
GLOBAL COMMENT: It's unfortunate that prettier works this way on chai assertions, as the `.property()` changes the assertion target, and the community convention is additional indentations after one of those to help visually confirm that's that case.

No action needed here, just too bad prettier doesn't handle things like chai assertions with "let the author actually decide"